doc/source/release-notes/Percona-Server-8.0.27-18.rst

```diff
@@ -0,0 +1,70 @@
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated. Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
+* Fix for when a parent table initiates a cascading ``SET NULL`` operation on the child table, the virtual column can be set to NULL instead of the value derived from the parent table.
+
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.27 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+
+
+New Features
+================================================================================
+
+* :jirabug:`PS-7960`: Document the ``rocksdb_partial_index_sort_max_mem`` variable, the ``rocksdb_bulk_load_partial_index`` variable, and the ``rocksdb_cancel_manual_compactions`` variable.
+
+
+
+Improvements
+================================================================================
+
+* :jirabug:`PS-7955`: Enable ZenFS functionality on standard Percona Server packages on Debian 11 and Ubuntu 20.04.
+* :jirabug:`PS-7931`: Implement a Slow Query Log rotation in Percona Server for MySQL 8.0.
+* :jirabug:`PS-6730`: In the Slow Query Log, the 'Last_errno:' field should only be used for errors.
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PS-7883`: An ``ALTER`` query when ``--rocksdb_write_disable_wal`` is enabled causes a server exit.
+* :jirabug:`PS-8007`: *Percona Server for MySQL* can also fail to start if the server starts before the network mounts the datadir or local mount of the datadir.
+* :jirabug:`PS-7977`: A broken AppArmor profile after an 8.0.22-13 to 8.0.23-14 upgrade. (Thanks to Alex Kompel for reporting this issue)
+* :jirabug:`PS-2346`: LP #720547: Implement an option to allow queries with specific error codes to add entries to the Slow Query Log.
+* :jirabug:`PS-7958`: A server exit when a ``SELECT`` statement uses a Full-Text Search index with special character.
+* :jirabug:`PS-7940`: Initializing a virtual column template if a child table has a virtual column. This initialization prevents a server exit with an ``ON DELETE CASCADE`` statement in the parent table and the child table had virtual columns. (Upstream :mysqlbug:`105290`)
+* :jirabug:`PS-5654`: On a view, the query digest for each SELECT statement is now based on the SELECT statement and not the view definition, which was the case for earlier versions (Upstream :mysqlbug:`89559`)
+* :jirabug:`PS-7975`: Fix to allow test main.mtr_unit_tests to complete successfully (Thanks to Thomas Deutschmann for reporting this issue)
+* :jirabug:`PS-7873`: Fix for when the log_status table reported an incorrect  executed_gtid (Thanks to zhujzhuo for reporting this issue) (Upstream :mysqlbug:`102175`)
+* :jirabug:`PS-5168 Fix for when the Slow Query Log reports ``tmp_table_size:0``. 
```

Comment:
Fix for when the Slow Query Log reports - The title is a link now

Reply:
Unclear.

Reply:
`PS-5168`:

Reply:
Thanks.